test/test.py

```diff
@@ -72,7 +72,7 @@ def exe(command):
 # ------------------------------------------------------------------------------
 
 
-def configure_build_and_exe(name, setup_command):
+def configure_build_and_exe(name, setup_command, make_command = None, run_command = None):
```

Comment:
I had to modify this Python function to incorporate user's defined make and run commands.